lib/options.py

```diff
@@ -201,135 +286,190 @@ def __init__(self):
                 default=None,
                 type=os.path.abspath,
-                help="""Path to snapshot that will be loaded before testing.""")
+                help=format_help(
+                    """
+                    Path to snapshot that will be loaded before testing.
+                    """))
 
         parser.add_argument(
                 "--disable-schema-upgrade",
                 dest='disable_schema_upgrade',
                 action="store_true",
                 default=False,
-                help="""Disable schema upgrade on testing with snapshots.""")
+                help=format_help(
+                    """
+                    Disable schema upgrade on testing with snapshots.
+                    """))
+
         parser.add_argument(
                 "--memtx-allocator",
                 dest="memtx_allocator",
                 default=os.environ.get("MEMTX_ALLOCATOR", "small"),
-                help="""Memtx allocator type for tests""")
+                help=format_help(
+                    """
+                    Memtx allocator type for tests.
+                    """))
+
         parser.add_argument(
                 '--tags',
                 dest='tags',
                 const=self._show_tags,
                 nargs='?',
                 default=None,
                 type=split_list,
-                help="""Comma separated list of tags.
+                help=format_help(
+                    """
+                    Comma separated list of tags.
+
+                    If tags are provided, test-run will run only those tests,
+                    which are marked with ANY of the provided tags.
 
-                If tags are provided, test-run will run only those
-                tests, which are marked with ANY of the provided
-                tags.
+                    If the option is given without a parameter (at the last
+                    position), test-run will show a list of tags and stop.
+                    """))
 
-                If the option is given without a parameter (at the
-                last position), test-run will show a list of tags
-                and stop.
-                """)
         parser.add_argument(
                 '--env',
                 dest='show_env',
                 action='store_true',
                 default=False,
-                help="""Print environment variables, which are set by test-run.
+                help=format_help(
+                    """
+                    Print environment variables, which are set by test-run.
 
-                Useful for, say, running built-in luatest executable.
+                    Useful for, say, running built-in luatest executable.
 
-                Usage: just source it into a current environment:
+                    Usage: just source it into a current environment:
 
-                ```
-                . <(./test/test-run.py --env)
-                ```
-                """)
+                    ```
```

Comment:
Why not just `$ . <(./test/test-run.py --env)`?

Reply:
It feels for me that I should somehow mark a code block. It is not rendered anyhow specific in the `/path/to/test-run.py --help` output, but Markdown is so widespread that I found it acceptable to use it in the plain text for this purpose.

It has not much meaning here, more a matter of taste. I dropped the lines with backticks.